packages/components/Spinner/ios/MSFSpinnerViewManager.m

```diff
@@ -0,0 +1,42 @@
+#import <React/RCTComponent.h>
+#import <React/RCTViewManager.h>
+
+@import FluentUI;
+
+@interface MSFActivityIndicatorView(RCTComponent)
+
+@end
+
+@implementation RCTConvert (MSFActivityIndicatorView)
+
+RCT_ENUM_CONVERTER(MSFActivityIndicatorViewSize, (@{
+	@"xSmall": @(MSFActivityIndicatorViewSizeXSmall),
+	@"small": @(MSFActivityIndicatorViewSizeSmall),
+	@"medium": @(MSFActivityIndicatorViewSizeMedium),
+	@"large": @(MSFActivityIndicatorViewSizeLarge),
+	@"xLarge": @(MSFActivityIndicatorViewSizeXLarge),
+
+}), MSFActivityIndicatorViewSizeSmall, integerValue);
+
+@end
+
+@interface RCT_EXTERN_MODULE(MSFSpinnerViewManager, RCTViewManager)
+
+RCT_EXPORT_VIEW_PROPERTY(hidesWhenStopped, BOOL);
+
+RCT_EXPORT_VIEW_PROPERTY(color, UIColor);
+RCT_EXPORT_VIEW_PROPERTY(size, MSFActivityIndicatorViewSize);
+
+RCT_CUSTOM_VIEW_PROPERTY(animating, BOOL, MSFActivityIndicatorView)
```

Comment:
it seems weird that changing the value is going to start or stop the animation. or am I interpreting wrong?

Reply:
It does. I copied this implementation from fabric-react-native. I think that was done to match the API of the base react-native activity-indicator https://reactnative.dev/docs/activityindicator